packages/block-library/src/navigation/edit/index.js

```diff
@@ -310,6 +319,7 @@ function Navigation( {
 				'is-vertical': orientation === 'vertical',
 				'no-wrap': flexWrap === 'nowrap',
 				'is-responsive': isResponsive,
+				'is-collapsed': isCollapsed(),
```

Comment:
```suggestion
				'is-collapsed': isCollapsed,
```

Can this become a variable reference?